pkg/workspace/doc_extractor.go

```diff
@@ -6,21 +6,21 @@ package workspace
 import (
 	"fmt"
 
-	"github.com/k14s/ytt/pkg/structmeta"
+	"github.com/k14s/ytt/pkg/template"
 	"github.com/k14s/ytt/pkg/yamlmeta"
 	"github.com/k14s/ytt/pkg/yamltemplate"
 )
 
 const (
-	AnnotationDataValues       structmeta.AnnotationName = "data/values"
-	AnnotationDataValuesSchema structmeta.AnnotationName = "data/values-schema"
+	AnnotationDataValues  template.AnnotationName = "data/values"
```

Comment:
not formatted. im guessing you are not running ./hack/build.sh?

Reply:
ah good catch! I do, but this was due to resolving a merge conflict (and then me not running formatting again)